docs/_posts/2018-04-23-moving-to-modern-cpp.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: blogpost
+title:  "Open GEE moves to modern C++"
+section_id: blog
+date:   2018-04-23 13:30:00
+author: Open GEE Development Team
+---
+
+Hello Everyone!
+
+The Open GEE Development Team would like to make a special announcement about an
+effort to upgrade our usage of `C++` to a more modern standard. As of release
+`Open GEE 5.2.3`, we will begin compiling Open GEE code using the `C++11` standard.
+
+`C++11` was a major new release of `C++` with many new features to help with
+better memory management, performance, and developer productivity. We want working
```

Comment:
may be rephrase "We want working" ... may be some thing like "We want the development experience on ..." or similar